core/src/main/scala/org/apache/spark/scheduler/TaskSchedulerImpl.scala

```diff
@@ -275,9 +275,9 @@ private[spark] class TaskSchedulerImpl(
       } while (launchedTask)
     }
 
-    while(!serializingTask.isEmpty) {
-      Thread.sleep(1000)
-    }
+    do {
+      Thread.sleep(1)
+    } while(!serializingTask.isEmpty) 
```

Comment:
excuse me, why did you change here? I cannot understand what's the difference @qqsun8819 (may be you have forget it,hahhh